### ISSUE TITLE
@itzJOKINHA

### ISSUE THREAD

**Issue by oOBlueArrowOo** [finndev/PokeBuddy#373]
u got ripped off i got it for free xD

**Comment by Rab89** [comment]
is there any need to create an issue specifically for this??

**Comment by curtiola** [comment]
No, there are already a crap ton of shit posts for vanethir to filter through in the next little while
